journal/202306.md

```diff
@@ -0,0 +1,861 @@
+
+Like all previous consensus changes, the consensus code will remain dormant and only activate if the respective [voting agendas](https://voting.decred.org/) pass.
+
+The following work has been merged in `master` towards future releases:
+
+- Updated to use latest [dcrtest](https://github.com/decred/dcrd/pull/3142), and excluded [unused modules](https://github.com/decred/dcrd/pull/3143) from regular builds.
+- Updated the [Docker image](https://github.com/decred/dcrd/pull/3146) to build the dcrd binary with newer Go and Alpine Linux. It's noteworthy to mention that the dcrd Docker template does not persist the Alpine Linux OS. After building the binary, the OS is removed, and dcrd runs in a `scratch` Docker container by itself. No OS, no root user, not even a shell. A lightweight [container with maximum security](https://github.com/decred/dcrd/tree/4aa67947b5b36d40bb146d5ecacb76dcb7f50a7f/contrib/docker).
+- Added [release notes for v1.8.0](https://github.com/decred/dcrd/pull/3144).
+- Cleaned up [database test logic](https://github.com/decred/dcrd/pull/3147).
+- Added [release notes templates](https://github.com/decred/dcrd/pull/3148).
+- Added [CI support](https://github.com/decred/dcrd/pull/3145) for test and module caches, which will result in faster CI runs. CI is Continuous Integration, a service that *continuously* builds and tests the code to ensure all changes *integrate* well. Faster CI runs reduce developers' wait time to see if their changes pass all tests. The cache cuts CI run time from ~10 to ~1 minute.
+- Updated [module hierarchy diagram](https://github.com/decred/dcrd/pull/3149) to accurately reflect module dependencies.
+- [Improved test flag handling](https://github.com/decred/dcrd/pull/3151), especially in cases where default home directories are inaccessible (such as with NixOS) and need to be changed. Additionally, a [`DCRD_APPDATA`](https://github.com/decred/dcrd/pull/3152) environment variable can now be used to override the default directory. Environment variables are generally the preferred approach for overriding defaults in containers.
+
+Changes merged in the [dcrtest](https://github.com/decred/dcrtest) framework:
+
+- Updated the voting wallet to support [DCP-12](https://github.com/decred/dcrtest/pull/12) block reward split semantics on simnet.
+- Updated the voting wallet to support [DCP-11](https://github.com/decred/dcrtest/pull/14) proof of work semantics on simnet.
+- Updated to use latest released [Decred module](https://github.com/decred/dcrtest/pull/15) dependencies and latest [main dcrd module](https://github.com/decred/dcrtest/pull/16).
+
+{ _omitted minor PRS: 3139, 3140, 3141, dcrtest#13_ }
+
+{ _image candidate: [Initial Sync Total Bytes Used Comparison](https://matrix.to/#/!zefvTnlxYHPKvJMThI:decred.org/$mbJD22XpiecybtWfRIhFRTawODG9wQujjoKSyOqegI0) - what are these "Total Bytes"? memory?_ }
```

Comment:
It's the total number of bytes allocated in order to perform the initial sync.  Less bytes allocated means less work for the GC and, by extension, less overall work for the CPU which equates to faster sync times.

Reply:
Got it, thanks. Will use that in the image caption.